File: FrontEnd/src/modules/Trader/models/AccountInfoView.ts
export type TokenStatus = "empty" | "invalid" | "expired" | "valid";

export interface AccountInfoView {
  id: string;
   broker: number;  
  name: string;
  username: string;
  
  sessionStatus: "empty" | "valid" | "expired" | "invalid";  // ← rename
  sessionExp?: number | null;        // ← rename از tokenExp
}

File: Trader.Application/Dtos/AccountInfoView.cs
namespace Trader.Application.Dtos
{
    public class AccountInfoView
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = default!;
        public string Username { get; set; } = default!;

        /// <summary>"empty" | "valid" | "expired" | "invalid"</summary>
        public string TokenStatus { get; set; } = "empty";

        public string? TokenExp { get; set; }   // ISO 8601
    }
}